README.md

```diff
@@ -1 +1,3 @@
-# CSCI-336-2023-Rafael-Castillo
\ No newline at end of file
+# CSCI-336-2023-Rafael-Castillo
+
```

Comment:
Q-*/F-*: Your file changes for the assignment don't actually appear in this PR:
 - Q-3: Some funny formatting and indentation in your files. 
 - Q-4: The file/folder structure isn't what it's supposed to be.
 - F-1: You are calling the /jokes/random endpoint instead of /jokes.
 - Q-5: The public endpoints do not need an api key.
 - F-2: You didn't follow the responsive specifications for the rows/cols.